app/javascript/tests/util/DeleteDialog.test.js

```diff
@@ -131,4 +131,14 @@ describe('DeleteDialog', () => {
     expect(wrapper.state('loading')).toBeTruthy();
     expect(wrapper.find(Modal.Footer).find(Button).at(1).prop('disabled')).toBeTruthy();
   });
+
+  it('Prompts the user for symptom onset if necessary', () => {
+    const wrapper = shallow(<DeleteDialog type={mockType} delete={deleteMock} toggle={toggleMock} onChange={onChangeMock} showSymptomOnsetInput={true} />);
+    expect(wrapper.find('#symptom_onset_delete_dialog').exists()).toBeTruthy();
```

Comment:
Can you check that the alert and label are properly rendered here as well? we don't check that elsewhere

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/a5f7d0708f903826cb1f4dec5169309b37c77dbb#diff-640be94f511d6918e9e1e33925a272262c842e30e77a89b05a1555ee5dae352cR137